suit_generator/cli.py

```diff
@@ -28,9 +33,36 @@
 }
 
 
+def configure_cli_logging(log_file_name: str = None):
+    """
+    Configure logging for CLI.
+
+    :param log_file_name: log file name to be used (ovveride default log file name from logging.yaml)
+    """
+    dir_path = pathlib.Path(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
I suggest to not mix os.path and pathlib.Path in the same module, generally in recent versions of python standard libraries pathlib.Path should contain all the needed functionality that was before in os.path. In this case I think it should be `dir_path = Path(__file__).resolve().parent`

Reply:
it was in __init__ file - I've just moved it - fixed